packages/phrases/src/locales/en.ts

```diff
@@ -27,8 +27,20 @@ const errors = {
   },
   oidc: {
     aborted: 'The end-user aborted interaction.',
-    invalid_scope: 'Scope {{scopes}} is not supported.',
+    invalid_scope: 'Scope {{scope}} is not supported.',
     invalid_scope_plural: 'Scope {{scopes}} are not supported.',
+    invalid_token: 'Invalid token provided.',
+    invalid_client_metadata: 'Invalid client metadata provided.',
+    insufficient_scope: 'Access token missing requested scope {{scopes}}',
+    invalid_request: 'Request is invalid',
+    invalid_grant: 'Grant request is invalid',
+    invalid_redirect_uri: "Redirect_uri did not match any of the client's registered redirect_uris",
+    access_denied: 'Access denied',
```

Comment:
should have the period for all phrases since `message` implies a full sentence.

```suggestion
    access_denied: 'Access denied.',
```